src/Core/src/Graphics/IShapeView.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace Microsoft.Maui.Graphics
+{
+	/// <summary>
+	/// Represents a View that enables you to draw a shape to the screen.
+	/// </summary>
+	public interface IShapeView : IView
+	{
+		/// <summary>
+		/// Gets the Shape definition to render.
+		/// </summary>
+		Shape Shape { get; }
+
+		/// <summary>
+		/// Indicates the brush used to paint the shape's interior.
+		/// </summary>
+		Color Fill { get; }
```

Comment:
This must be a Brush. Ideally, we merge the Brushes PR first and after that we update this PR or else, we create another PR where we change this `Color` for a `Brush`.